server/integration/event-logger/src/main/java/org/infinispan/server/eventlogger/ServerEventImpl.java

```diff
@@ -20,17 +14,24 @@
  * @author Tristan Tarrant
  * @since 8.2
  */
-@SerializeWith(ServerEventImpl.Externalizer.class)
 public class ServerEventImpl implements EventLog {
 
-   private final EventLogLevel level;
-   private final EventLogCategory category;
-   private final Instant when;
-   private final String message;
-   private final Optional<String> detail;
-   private final Optional<String> context;
-   private final Optional<String> who;
-   private final Optional<String> scope;
+   @ProtoField(number = 1, required = true)
```

Comment:
For backwards compatibility we shouldn't make anything required, IIRC the next version of Protobuf won't have required fields at all.